lib/createTopLevelExpect.js

```diff
@@ -1453,12 +1490,49 @@ expectPrototype._executeExpect = function (
   return oathbreaker(assertionRule.handler(wrappedExpect, subject, ...args));
 };
 
+expectPrototype._camelCaser = function (context, subjectType, subject) {
+  const methods = {};
+  let instance = this;
+  while (instance) {
+    Object.keys(instance.assertions).forEach((testDescriptionString) => {
+      if (
+        !subjectType ||
+        instance.assertions[testDescriptionString].some((assertion) =>
+          subjectType.is(assertion.subject.type)
+        )
+      ) {
+        let method = (subject) => (...args) => {
+          return this._expect(context, [
+            subject,
+            testDescriptionString,
+            ...args,
+          ]);
+        };
+
+        if (subjectType) {
+          method = method(subject);
+        }
+
+        methods[
+          testDescriptionString.replace(/ [a-z]/g, ($0) =>
+            $0.charAt(1).toUpperCase()
+          )
+        ] = method;
+      }
+    });
+    instance = instance.parent;
+  }
+  return methods;
+};
+
 expectPrototype._expect = function (context, args, forwardedFlags) {
   const subject = args[0];
   const testDescriptionString = args[1];
 
-  if (args.length < 2) {
-    throw new Error('The expect function requires at least two parameters.');
+  if (args.length < 1) {
+    throw new Error('The expect function requires at least one parameter.');
+  } else if (args.length === 1) {
+    return this._camelCaser(context, this.findTypeOf(subject), subject);
```

Comment:
Re the latter part of my comment [here](https://github.com/unexpectedjs/unexpected/pull/784#issuecomment-753994990), it's weakening the two parameter rule and making a single argument suddenly make the library behave very different that got me pondering whether this bends things a little too far.

Reply:
I don't think it's too bad -- it actually aligns the one parameter expect case with what the neste `expect` made available to assertion rule handlers does: https://github.com/unexpectedjs/unexpected/blob/55268ea8c80cc34b527bd114b3e516807e1f3244/lib/createTopLevelExpect.js#L1313

... And if we agree about a long term goal of deprecating/removing the old string-based syntax, I think we can live with a bit of temporary overloading.